README.md

```diff
@@ -129,6 +129,58 @@ This is ideal if you are worried about accidentally exposing your metrics, are c
 
 The exporter runs by default on port `9394` and the metrics are available at the standard path of `/metrics`, the stand-alone exporter is configured to use gzip.
 
+
+### Rack Middleware
+
+Promenade provides custom Rack middleware to track HTTP response times for requests in your Rack application.
+
+This was originally inspired by [prometheus-client-mmap](https://gitlab.com/gitlab-org/prometheus-client-mmap/-/blob/master/lib/prometheus/client/rack/collector.rb).
+
+**This middleware is automatically added to your Rack stack if your application is a Ruby on Rails app.**
```

Comment:
```suggestion
**This middleware is automatically added to your [middleware stack](https://guides.rubyonrails.org/rails_on_rack.html#inspecting-middleware-stack)**
```

Reply:
I think we want to make it clear that this only happens when the app is a Rails app though?

Reply:
correct